src/cloud/cdn/configure-fastly-customize-cache.md

```diff
@@ -0,0 +1,226 @@
+As a result, you could receive 503 errors when attempting operations that require lengthy processing, or when trying to perform bulk operations.
+
+If you need to complete bulk actions that take longer than 3 minutes, you can prevent 503 errors by changing the _Admin path timeout_ value.
+
+{:.procedure}
+To extend the Fastly timeout for the Magento Admin:
+
+{% include cloud/admin-ui-login-step.md %}
+
+1. Click **Stores** > Settings > **Configuration** > **Advanced** > **System** and expand **Full Page Cache**.
+
+1. In the *Fastly Configuration* section, expand **Advanced**.
+
+1. Set the **Admin path timeout** value in seconds. This value cannot be more than 10 minutes (600 seconds).
+
+1. Click **Save Config** at the top of the page.
+
+1. After the page reloads, click **Upload VCL to Fastly** in the *Fastly Configuration* section.
+
+Fastly gets the Magento Admin path for generating the VCL file from the `app/etc/env.php` configuration file.
+
+## Configure purge options
+
+Fastly provides multiple types of purge options on your Magento Cache Management page, including purging product category, product assets, and content. When enabled, Fastly watches for events to automatically purge those caches. If you disable a purge option, you can manually purge Fastly caches after finishing updates through the Cache Management page.
+
+The options include:
+
+-  **Purge category**–Purges product category content (not product content) when you add and update a single product. You may want to keep this disabled and enable purge product, which purges products and product categories.
+-  **Purge product**–Purges all product and product category content when saving a single modification to a product. Enabling purge product can be helpful to immediately get updates to customers when changing a price, adding a product option, and when product inventory is out-of-stock.
+-  **Purge CMS page**–Purges page content when updating and adding pages to the Magento CMS. For example, you may want to purge when updating your Terms and Conditions or Return policy. If you rarely make these changes, you could disable automatic purging.
+-  **Soft purge**–Sets changed content to stale and purges according to the stale timing. In combination with the stale timings your customers will be served stale content very fast while Fastly is updating the content in the background.
+
+![Configure purge options]({{ site.baseurl }}/common/images/cloud/cloud_fastly-purgeoptions.png){:width="650px"}
+
+{:.procedure}
+To configure Fastly purge options:
+
+1. In the *Fastly Configuration* section, expand **Advanced**.
+
+1. All purge options display. Select "Yes" per purge option to enable automatic purging. Select "No" to disable automatic purging, allowing you to manually purge caches through the Cache Management page.
```

Comment:
"per purge option" I wonder if the use of **per** will be confusing to some readers. Maybe use "for each" or something similar?